public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```

Comment:
Where does this rule apply?

Reply:
I have no idea about this file. This is default file from create-react-app